app/web/components/Navigation/ReportButton.tsx

```diff
@@ -169,9 +170,11 @@ export default function ReportButton({
                 {t("report.bug.button_label")}
               </StyledReportTypeButton>
               <StyledReportTypeButton
-                onClick={() => {
-                  setType("content");
-                }}
+                // href={helpCenterReportContentURL}
+                onClick={() =>
+                  (window.location.href = helpCenterReportContentURL)
+                }
+                component="button"
```

Comment:
I wouldn't adjust the code to the test, but rather update the test to account for the new code here. I'll leave a suggestion in the test below!